src/megatron/bridge/models/deepseek/common.py

```diff
@@ -116,6 +117,8 @@ def get_common_mapping_list() -> list:
         "decoder.layers.*.self_attention.linear_q_proj.weight": "model.layers.*.self_attn.q_proj.weight",
     }
 
+    add_separate_layernorm_mappings(param_mappings)
```

Comment:
can we do this in MappingRegistry init and properly comment it? it looks general enough to put there instead ask developer to do special things here. It will change the logic a bit, since in MappingRegistry init you are receiving a list of mappings not dict.